CONTRIBUTING.md

```diff
@@ -1,25 +1,114 @@
 # Overview
 
-This documents explains the processes and practices recommended for contributing enhancements to the LMA Light bundle.
+This documents explains the processes and practices recommended for
+contributing enhancements to the LMA Light bundle.
 
-- Generally, before developing enhancements to this charm, you should consider [opening an issue](https://github.com/canonical/lma-light-bundle) explaining your use case.
-- If you would like to chat with us about your use cases or proposed implementation, you can reach us on the [Charmhub Mattermost](https://chat.charmhub.io/charmhub/channels/charm-dev) or [Discourse](https://discourse.charmhub.io/).
+- Generally, before developing enhancements to this charm, you should consider
+  [opening an issue](https://github.com/canonical/lma-light-bundle) explaining
+  your use case.
+- If you would like to chat with us about your use cases or proposed
+  implementation, you can reach us on the
+  [Charmhub Mattermost](https://chat.charmhub.io/charmhub/channels/charm-dev)
+  or [Discourse](https://discourse.charmhub.io/).
 - All enhancements require review before being merged.
-  Besides the code quality and test coverage, the review will also take into  account the resulting user experience for Juju administrators using this charm.
-  Please simplify the code-review process by rebasing onto the `main` branch and avoid merge commits so we can enjoy a linear Git history!
+  Apart from code quality and test coverage, the review will also take into
+  account the Juju administrator user experience using the bundle.
 
 ## Development
 
-### Deploy from a local bundle file
+There are four charms used for development:
+- prometheus
+- alertmanager
+- grafana
+- tester
+
+### Deploy with local charms
+
+To deploy the bundle using only/some local charms you need to render the
+[`bundle.yaml.j2`](bundle.yaml.j2) template
+and then deploy the rendered bundle as usual.
+
+#### Render template using the rendering script
+You can render and deploy a production bundle using:
 
 ```shell
+# generate and activate a virtual environment with dependencies
+tox -e integration --notest
+source .tox/integration/bin/activate
```

Comment:
I would maybe just replace this with `pip install jinja2` No need to install a whole bunch of libraries just to build it.